### DISCUSSION TITLE
In the skywalking topology, the service name is too long and becomes an ellipsis. How to display the complete service name

### DISCUSSION THREAD

**Discussion by czxin788** [apache/skywalking#8688] [Q&A] [answered]
In the skywalking topology, the service name is too long and becomes an ellipsis. How to display the complete service name.

![image](https://user-images.githubusercontent.com/13861904/158527333-212f5675-7f06-43b1-b450-afc7bf7836a5.png)

**Comment by wu-sheng** [comment, marked as answer]
For now, you can't.

**Comment by czxin788** [comment]
Is it support in the future?
I think it is nesessary to display the complete service name,otherwise，we  only see the group name for every one in the topology

**Reply by wu-sheng** [reply]
In the SkyWalking community, feature/enhancement happens if someone contributes. So, I don't know.
We are welcome you or someone to do this.

**Reply by czxin788** [reply]
thank you for reply

**Comment by czxin788** [comment]
if the group name is renamed, how to delete the old group name?

![image](https://user-images.githubusercontent.com/13861904/158544474-29633dc0-761b-4022-bbba-5ac6dc2a1791.png)
